.github/workflows/verify.yml

```diff
@@ -8,10 +8,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Use Node.js 12.x
```

Comment:
I would go with 14 because [is the current active branch in Node releases lifecyle](https://nodejs.org/en/about/releases/#releases). It would be also awesome to test in different versions in master or periodically, but not for all PRs.

Reply:
Good point. I am going to close this pull request and take more time to research a better solution for this.

Thank you so much.

Reply:
To be more precise, v14 is the active LTS whereas v16 is the active "branch". The relevant difference between v14 and v16 for us, as mentioned in the other comment, is the change from NPM v6 to NPM v7 by default (though you can upgrade to NPM v7 before Node v16) which comes with a change to the `package-lock.json` file format.

To avoid confusion like this, I think it might be best to upgrade to Node v16/NPM v7. Even though prior NPM versions don't fully support the new format, it's still able to install the dependencies in most cases (I'm not sure on the details but I'm pretty sure it won't matter much to this project). Importantly, NPM prior to v7 won't downgrade the `package-lock.json` to v1, whereas NPM v7 will upgrade it to v2.

In any case, this transition period from NPM v6 to NPM v7 is a bit of a mess and I'm also not sure what the best way to deal with it is. So in that regard, thanks for bringing this up for dicussion @LitoMore :smile: 


> Consistent with the Node.js version in Docker.

To this point, I don't think it's necessary to align the versions "just because". The reason the Dockerfile has v16 is because it got updated automatically by Renovate and since it's just intended for running SVGO in a Docker container I didn't bother telling Renovate to stop updating it. So, unless someone actually wants to use the Dockerfile for development (if that's you @LitoMore, please say so) I don't see a reason spend time aligning things with the Dockerfile.

Reply:
@ericcornelissen Agree! Thank you for your answer!

> I don't think it's necessary to align the versions "just because".

I just noticed the Docker is using Node.js 16, but I didn't realize the Docker is not for the development.

Reply:
> I just noticed the Docker is using Node.js 16, but I didn't realize the Docker is not for the development.

The confusion is understandable :slightly_smiling_face:  just wanted to clarify.